### DISCUSSION TITLE
Multiple addresses

### DISCUSSION THREAD

**Discussion by FilipDem** [jvanderzande/GarbageCalendar#33] [Ideas]
Hi Jos,
I have 2 different addresses. Is there any way to get both supported without the need to duplicate the scripts?
I am from Belgium and both addresses are supported by the "recycleapp.be"... 
Thanks for your advice.
Filip

**Comment by jvanderzande** [comment]
That won't be a simple task to change as they will possibly needs their own unique configs with all parameters and garbagetype_cfg table. 
Doubt many would ever have this requirement and running it with 2 separate scripts, each in their own subdirectory from a script_time_runboth.lua , is way simpler to get this done. :)
